.github/workflows/pr-release.yml

```diff
@@ -0,0 +1,44 @@
+name: PR-Release
+
+on:
+  pull_request:
+    types: [issue_comment]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  changeset:
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'release/next') && github.event.pull_request.merged == false && contains(github.event.comment.body,'/publish') && github.event.comment.author_association == 'MEMBER' }}
```

Comment:
what was the thought behind requiring label **and** a comment?

Seems like we could maybe remove the label requirement

Reply:
the label I thought would help in organizing and keeping track of the PRs currently published. 
But think you are right I will remove that check and let the maintainer decide on the label